src/Sri.php

```diff
@@ -8,15 +8,22 @@ public function __construct()
     {
     }
 
-    public function html(string $path): string
+    public function html(string $path, bool $with_credentials = false): string
```

Comment:
`$with_credentials` should be in camelCase to stay consistent with the rest of the code.